src/main/resources/sql/data.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO `user` (`id`, `login`, `email`, `password`, `created_at`) VALUES (1, 'johny', 'j.ter@gmail.com', 'afgadh1334FDSADS', CURRENT_TIMESTAMP);
```

Comment:
Does it get populated by H2?

Reply:
Yes, but more inserts will be in another task until everything is fine-filled